pkg/generator/templates/cloud-init.gardenlinux.template

```diff
@@ -74,6 +74,9 @@ fi
 systemctl enable docker && systemctl restart docker
 {{- end }}
 
+{{- $bootstrap := .Bootstrap -}}
 {{- range $_, $unit := .Units }}
+{{- if or (ne $unit.Name "cloud-config-downloader.service") ($bootstrap) }}
 systemctl enable '{{ $unit.Name }}' && systemctl restart '{{ $unit.Name }}'
 {{- end }}
+{{- end }}
```

Comment:
Hm, out of curiosity, does this not work? 

```suggestion
{{- range $_, $unit := .Units }}
{{- if or (ne $unit.Name "cloud-config-downloader.service") ($.Bootstrap) }}
systemctl enable '{{ $unit.Name }}' && systemctl restart '{{ $unit.Name }}'
{{- end }}
{{- end }}
```

Reply:
It works.

Reply:
Cool. Maybe you want to do the same in the other PRs